src/map.h

```diff
@@ -1228,9 +1220,6 @@ class map
         // TODO: fix point types (remove the first overload)
         bool has_flag_furn( ter_furn_flag flag, const tripoint &p ) const;
         bool has_flag_furn( ter_furn_flag flag, const tripoint_bub_ms &p ) const;
-        bool has_flag_furn( ter_furn_flag flag, const point &p ) const {
-            return has_flag_furn( flag, tripoint( p, abs_sub.z() ) );
-        }
```

Comment:
There are a few `point` overloads like this that you simply removed instead of making a `point_bub_ms` version. Is there a particular reason for that? I didn't check all of them, but they seemed to have been rarely used anyway, so I don't think it's a real issue, I'm just wondering.

Reply:
I removed them when they weren't used anyway. My thinking was that it's better to try to use 3D coordinates than to use 2D with an implicit Z coordinate coming in from the side, so removing unused operations pushes future use towards the existing 3D versions.

It's not a clear cut and obvious path, though, so I'd like to hear if you think it's misguided.

Reply:
I'm not deep enough into map code to really have a preference there, so I'm fine with it either way.